.github/workflows/linter.yaml

```diff
@@ -31,3 +31,14 @@ jobs:
 
           # Optional: if set to true then the action don't cache or restore ~/.cache/go-build.
           # skip-build-cache: true
+  gosec:
```

Comment:
what level of security issue makes the whole check red?

Reply:
I think these just might be caused the fact that you're using the latest greatest version of the github action, while our lint task pins the version of golang-lint. What about trying to change this https://github.com/redhat-appstudio/service-provider-integration-operator/blob/315765437588437b7cc9fb2a9ce1efd6c7764ecd/.github/workflows/linter.yaml#L18 to `latest` and see if we get the same new warnings?